pkg/cmd/server/origin/run_components.go

```diff
@@ -349,6 +351,22 @@ func (c *MasterConfig) RunImageImportController() {
 	controller.Run()
 }
 
+// RunNetIDAllocationController starts the VNID allocation controller process.
+func (c *MasterConfig) RunNetIDAllocationController() {
+	// NetID allocator is only required for multitenant network plugin
+	if !sdnfactory.IsMultitenantNetworkPlugin(c.Options.NetworkConfig.NetworkPluginName) {
+		return
+	}
+
+	repair := vnidcontroller.NewRepair(15*time.Minute, c.MultitenantNetworkConfig.NetNamespaceRegistry, c.MultitenantNetworkConfig.NetIDRange, c.MultitenantNetworkConfig.NetIDRegistry)
```

Comment:
I see. I'd rather have a helper function on master config that builds the netNamespaceRegistry/netIDRange/netIDRegistry and call that function from here and in GetRestStorage(), rather than mutating the config in GetRestStorage().